js/main.js

```diff
@@ -4,22 +4,21 @@ var $input = document.querySelector('#photo');
 var $img = document.querySelector('.update-img');
 var $form = document.querySelector('form');
 var $imgSrc = './images/placeholder-image-square.jpg';
+var $ul = document.querySelector('ul');
 
 function handleEntry(event) {
   if ($input.value === '') {
     $img.src = $imgSrc;
   } else {
     $img.src = $input.value;
   }
-
 }
 $input.addEventListener('input', handleEntry);
 
 function handleSubmit(event) {
-  event.preventDefault();
```

Comment:
This should not have been removed, it is making your page refresh, which is not the right way to get the newly submitted entry visible immediately. Once an entry is submitted, you should generate a DOM tree for it and add that DOM tree to the beginning of the list in the `ul` element. There should be no refreshing necessary to make the app function.